en/admin/admin_utils.rst

```diff
@@ -790,7 +790,7 @@ Reference to the object deleted during compacting is displayed as **NULL**, whic
 
 *   *database_name*: The name of the database whose space is to be compacted. The path name to the directory where the database is to be created must not be included.
 
-*   *class_name_list*: You can specify the list of tables names that you want to compact space after a database name; the **-i** option cannot be used together. If you use the lists on client/server mode, It is skipped freeing objects such as catalog, delete files and tracker etc. 
+*   *class_name_list*: You can specify the list of tables names that you want to compact space after a database name; the **-i** option cannot be used together. If you use the lists on client/server mode, It skips securing space taken by objects such as catalog, delete files and tracker, etc. 
```

Comment:
'It' need to be changed to lower case 'it'

Reply:
thank you. i modified it.